examples/shopify/README.md

```diff
@@ -0,0 +1,201 @@
+# Demo
+
+[https://makeswift-examples-shopify.vercel.app](https://makeswift-examples-shopify.vercel.app)
+
+# Visually build with Shopify components
+
+In this example, you will learn how to integrate [Shopify](https://www.shopify.com/) with [Makeswift](https://www.makeswift.com) to create a visually editable ecommerce store.
+
+This example includes a home page for listing products by category and a product template page for including all the details.
+
+## Tools
+
+- [**Shopify StoreFrontAPI**](https://shopify.dev/api/storefront): to pull data related to a store in Shopify and provide it via React context.
+- [**Makeswift SDK**](https://www.makeswift.com/docs): to register components into Makeswift's visual builder.
+
+---
+
+## Using this repo
+
+1. **Create a site in Makeswift and a shop in Shopify**
+
+   Head over to [Makeswift](https://app.makeswift.com) and sign up for a free account. Create a site using the option to "Integrate with Next.js",
+   and copy the Site API key. You will need it in step 3.
+
+   Then head over to [Shopify](https://www.shopify.com/) and create a store. Here is [a good starting point](https://www.shopify.com/online) to get you started.
+
+2. **Clone this template**
+
+   Instead of using `create-next-app`, run this command from the terminal:
+
+   ```bash
+   npx degit makeswift/makeswift/examples/shopify  shopify
```

Comment:
There's an extra space between `makeswift/makeswift/examples/shopify` and `shopify`